pkg/cluster/purge_node.go

```diff
@@ -93,6 +93,18 @@ func (c *Controller) PurgeNode(ctx context.Context, name string, rook bool, rook
 			return err
 		}
 		c.Log.Infof("Purge node %q: deleted Kubernetes Node object", name)
+
+		rookMinHealthVersion, _ := semver.New("1.4.9")
+		if rookVersion.LT(*rookMinHealthVersion) {
+			c.Log.Warnf("The Rook version used is %s and it is recommdended update the Rook version. More info: https://kurl.sh/docs/install-with-kurl/managing-nodes#rook-ceph-cluster-prerequisites"+
+				" Be aware that by removing nodes with this Rook version it might let the Ceph unhealthy (knowing issue)."+
+				" Please, check the Ceph state and if you verify that it is unhealthy after remove the node then: \n"+
+				"- Stop the Rook operator (kubectl scale --replicas=0 deployment.apps/rook-ceph-operator -n rook-ceph) \n"+
+				"- Delete the deployment for the failed mon (kubectl -n rook-ceph get pod -l app=rook-ceph-mon) \n"+
+				"- Edit the configmap rook-ceph-mon-endpoints and (carefully) remove the failed mon from the list (kubect edit configmaps rook-ceph-mon-endpoints -n rook-ceph ) \n"+
+				"- Restart the operator (kubectl scale --replicas=1 deployment.apps/rook-ceph-operator -n rook-ceph) \n "+
+				"- Check the Rook operator logs and ensure that the ceph came back for a health state afterwords.", rookVersion)
```

Comment:
```suggestion
			c.Log.Warnf("The Rook version used is %s and it is recommended to update the Rook version. \n"+
				"More info: https://kurl.sh/docs/install-with-kurl/managing-nodes#rook-ceph-cluster-prerequisites \n"+
				"Be aware that by removing nodes with this Rook version it may leave Ceph unhealthy.\n"+
				"Please, check the Ceph status (kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status) \n"+
				"and if you verify that it is unhealthy after removing the node then: \n"+
				"- Stop the Rook operator (kubectl -n rook-ceph scale --replicas=0 deployment.apps/rook-ceph-operator) \n"+
				"- Delete the deployment for the failed mon (kubectl -n rook-ceph get pod -l app=rook-ceph-mon) \n"+
				"- Edit the configmap rook-ceph-mon-endpoints and (carefully) remove the failed mon from the list (kubect -n rook-ceph edit configmaps rook-ceph-mon-endpoints) \n"+
				"- Start the Rook operator (kubectl -n rook-ceph scale --replicas=1 deployment.apps/rook-ceph-operator) \n "+
				"- Ensure that Ceph came back in a healthy state (kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status) ", rookVersion)
```